frontend/src/components/Artist.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+export default function Artist() {
+  const { id } = useParams();
+  const [artist, setArtist] = useState(null);
+  useEffect(() => {
+    fetch(
+      `${
+        import.meta.env.VITE_BACKEND_URL ?? "http://localhost:5000"
+      }/artists/${id}`
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        setArtist(data);
+      });
+  }, []);
+  return (
+    <figcaption>
+      <p>{artist?.name}</p>
+      <p>{artist?.bio}</p>
+    </figcaption>
```

Comment:
a figcaption should be in a figure. I think what you want is:

    <article>
        <h1>{artist?.name}</h1>
        <p>{artist?.bio}</p>
    </article>